Framework/API/inc/MantidAPI/AlgorithmFactory.h

```diff
@@ -120,7 +126,7 @@ class MANTID_API_DLL AlgorithmFactoryImpl final
 
   /// Returns algorithm descriptors.
   std::vector<AlgorithmDescriptor>
-  getDescriptors(bool includeHidden = false) const;
+  getDescriptors(bool includeHidden = false, bool includeAliases = true) const;
```

Comment:
This is good that we can now control this and decide what we see. I think we should keep the default at `false` though so that it behaves as it did before so that current clients of this code don't get an unexpected change. Setting it to `false` will also ensure MantidPlot behaves as it used to without any changes.